### PR TITLE
sp-3 migrations

### PR DESCRIPTION
**Pull request by maybenull**

### CODE REVIEW

File: src/database/database.module.ts [hunk 1]
```diff
@@ -7,16 +7,7 @@ import { DatabaseService } from './database.service';
...
-    type: 'postgres',
-    host: configService.dbHost,
-    port: configService.dbPort,
-    username: configService.dbUser,
-    password: configService.dbPassword,
-    database: configService.dbName,
-    entities: ['src/**/**.entity.ts'],
-    synchronize: configService.dbSync, // todo: migrations to get db at right state w/o synchronize
-  });
+  return TypeOrmModule.forRoot();
```
Comment (stevespur) [inline src/database/database.module.ts]: does this get rid of the ConfigService?

File: .env.example [hunk 2]
```diff
@@ -10,4 +10,14 @@ DB_NAME=sp
...
\ No newline at end of file
+REDIS_PASSWORD=
+
+TYPEORM_CONNECTION=base
+TYPEORM_HOST=localhost
+TYPEORM_USERNAME=postgres
+TYPEORM_PASSWORD=secret
+TYPEORM_DATABASE=sp
+TYPEORM_PORT=5432
+TYPEORM_MIGRATIONS="migrations/*.js"
```
Comment (stevespur) [inline .env.example]: I feel like TYPEORM_MIGRATIONS, TYPEORM_MIGRATIONS_DIR, and TYPEORM_MIGRATIONS_TABLE_NAME shouldn't be configurable, that would imply different files per setup

File: src/database/database.module.ts (hunk 1 above)
Comment (maybenull) [reply src/database/database.module.ts]: i think it would

File: .env.example (hunk 2 above)
Comment (maybenull) [reply .env.example]: updated to `TYPEORM_MIGRATIONS="dist/migrations/*.js"`

i think the one with dist needs be configurable.  the others ones ya.  but i dont think its a huge issue

**Pull request merged**